src/banking_stage.rs

```diff
@@ -14,17 +14,18 @@ use service::Service;
 use std::net::SocketAddr;
 use std::sync::atomic::AtomicUsize;
 use std::sync::mpsc::{channel, Receiver, RecvTimeoutError, Sender};
-use std::sync::Arc;
+use std::sync::{Arc, Mutex};
 use std::thread::{self, Builder, JoinHandle};
 use std::time::Duration;
 use std::time::Instant;
 use timing;
 use transaction::Transaction;
 
+pub const NUM_THREADS: usize = 8;
```

Comment:
fixed number of threads?

Reply:
yea, for now.  we can make this dynamic later